tests/hybrid/test_layout.py

```diff
@@ -145,4 +145,33 @@ def test_hybrid_layout_solar_only(site):
 
     for i in range(4):
         assert solar_region[i] == pytest.approx(expected_solar_region[i], 1e-3)
-        assert buffer_region[i] == pytest.approx(expected_buffer_region[i], 1e-3)
\ No newline at end of file
+        assert buffer_region[i] == pytest.approx(expected_buffer_region[i], 1e-3)
+
+
+def test_kml_file_read():
+    filepath = Path(__file__).absolute().parent / "layout_example.kml"
+    SiteInfo.kml_read(filepath)
```

Comment:
Is this just testing that it doesn't raise an exception?

Reply:
The actual test is in line 159, but yes this line 153 will fail if an exception is raised. Is that what you mean?

Reply:
Oh nevermind, I see what you meant